webpack.config.js

```diff
@@ -1,22 +1,38 @@
 const path = require("path");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const ImageminPlugin = require("imagemin-webpack");
 const webpack = require("webpack");
 
 let config = {
     context: path.resolve(__dirname, "src"),
     entry: "./app.js",
     output: {
         filename: "bundle.js",
-        path: path.resolve(__dirname, "dist"),
+        path: path.resolve(__dirname, "dist")
     },
     plugins: [
         new CleanWebpackPlugin(),
         new CopyWebpackPlugin([{
```

Comment:
This should probably be replaced by proper injection at some point for cache busting. More info here: https://webpack.js.org/guides/output-management/